tools/release-scripts/release-1.5.sc

```diff
@@ -0,0 +1,177 @@
+import java.time.Instant
+
+import $ivy.`com.softwaremill.sttp::core:1.1.12`
+import $ivy.`com.typesafe.play::play-json:2.6.9`
+
+import $file.ui
+import $file.tokens
+import $file.changelog
+import $file.github
+import $file.jenkins
+import $file.context
+import context._
+import changelog.SemVer
+import ammonite.ops._
+import tokens.Credentials
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Promise}
+import ammonite.ops._
+
+type BranchName = String
+
+def checkIfShouldStartRelease(latestReleasedVersion: SemVer)(implicit buildContext: BuildContext) = {
+
+  implicit val path = buildContext.marathon.directory.path
+
+  val latestReleasedVersionString = latestReleasedVersion.toTagString()
+
+  val gitLogOutput = %%('git, 'log, s"$latestReleasedVersionString..releases/1.5").out.lines
+
+  if (gitLogOutput.map(_.trim).filter(_.nonEmpty).isEmpty) {
+    println("Nothing to release")
+    exit()
+  }
+}
+
+def commitAndPushChangelog(nextVersion: SemVer)(implicit context: BuildContext): BranchName = {
+  implicit val path = context.marathon.directory.path
+  val stringVersion = nextVersion.toReleaseString()
+  val branchName = s"changelog-update-$stringVersion"
+  %('git, 'checkout, "-b", branchName)
+  %('git, 'add, ".")
+  %('git, 'commit, "-m", s""" "changelog update for $stringVersion" """)
+  %('git, 'push, "origin", branchName)
+  branchName
+}
+
+def createAndPushTag(version: SemVer)(implicit buildContext: BuildContext): Unit = {
+  implicit val path = buildContext.marathon.directory.path
+  %('git, 'pull, "origin", "releases/1.5")
+  %('git, 'tag, "-a", version.toTagString(), "-m", version.toTagString())
```

Comment:
Why not use the Github API as done in [./ci/pipeline#315](https://github.com/mesosphere/marathon/blob/master/ci/pipeline#L315)?